app/main.ts

```diff
@@ -1,11 +1,38 @@
-import { bootstrap }    from '@angular/platform-browser-dynamic';
-import { HTTP_PROVIDERS } from '@angular/http';
-import {AUTH_PROVIDERS} from 'angular2-jwt';
+﻿import { bootstrap }    from '@angular/platform-browser-dynamic';
+import { HTTP_PROVIDERS, Http } from '@angular/http';
+import { AuthConfig, AuthHttp } from 'angular2-jwt';
 
 import { AppComponent } from './app.component';
 import { AuthGuard } from './shared/auth/auth-guard';
 import { AuthService } from './shared/auth/auth.service';
 import { ApiService } from './shared/api/api.service';
 import { HIP_ROUTER_PROVIDERS } from './app.routes';
+import { disableDeprecatedForms, provideForms } from '@angular/forms';
+import { provide } from '@angular/core';
 
-bootstrap(AppComponent, [HIP_ROUTER_PROVIDERS, HTTP_PROVIDERS, AUTH_PROVIDERS, AuthGuard, AuthService, ApiService]);
+bootstrap(AppComponent, [
+    disableDeprecatedForms(),
+    provideForms(),
+    HIP_ROUTER_PROVIDERS,
+    HTTP_PROVIDERS,
+    AuthGuard,
+    AuthService,
+    ApiService,
+    provide(
+        AuthHttp,
+        {
+            useFactory: (http: any) => {
+                return new AuthHttp(new AuthConfig({
+                    headerName: 'Authorization',
+                    headerPrefix: 'Bearer',
+                    tokenName: 'id_token',
+                    tokenGetter: (() => localStorage.getItem('id_token')),
+                    globalHeaders: [{ 'Content-Type': 'application/x-www-form-urlencoded' }],
+                    noJwtError: true,
+                    noTokenScheme: true
+                }), http);
+            },
+            deps: [Http]
+        })
+]);
+​
```

Comment:
__[tslint]__
* file should end with a newline

<!-- Comment by [SideCI](https://sideci.com) -->